spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -31,48 +31,109 @@ object SpdxLicenseAliasMapping {
      */
     private val customNames = mapOf(
         "AFLv2.1" to AFL_2_1,
+        "afl" to AFL_2_1,
+        "afl-2.0" to AFL_2_1,
         "ALv2" to APACHE_2_0,
         "APL2" to APACHE_2_0,
         "APLv2.0" to APACHE_2_0,
         "ASL" to APACHE_2_0,
         "Apache" to APACHE_2_0,
         "Apache-2" to APACHE_2_0,
         "Apache2" to APACHE_2_0,
+        "alv2" to APACHE_2_0,
+        "apache" to APACHE_2_0,
+        "apache_2_0" to APACHE_2_0,
+        "apache_v2" to APACHE_2_0,
+        "asl" to APACHE_2_0,
         "BOOST" to BSL_1_0,
-        "BSD" to BSD_2_CLAUSE,
+        "Bouncy" to MIT.toExpression(),
+        "bouncy" to MIT.toExpression(),
+        "bouncy-license" to MIT.toExpression(),
+        "BSD" to BSD_3_CLAUSE,
         "BSD-3" to BSD_3_CLAUSE,
+        "BSD*" to BSD_3_CLAUSE,
         "BSD-Style" to BSD_3_CLAUSE,
         "BSD-like" to BSD_3_CLAUSE,
         "BSD-style" to BSD_3_CLAUSE,
         "BSD2" to BSD_2_CLAUSE,
         "BSD3" to BSD_3_CLAUSE,
+        "bsd" to BSD_3_CLAUSE,
+        "bsd2" to BSD_2_CLAUSE,
+        "bsd3" to BSD_3_CLAUSE,
+        "bsl" to BSL_1_0,
+        "bsl-1.0" to BSL_1_0,
```

Comment:
This only differs in case to "BSL-1.0" and thus should not be needed. Similar for other entries.

Reply:
Fixed